.changeset/hip-seas-beam.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pantheon-systems/wordpress-kit": patch
```

Comment:
I see this one is a patch and the other is a minor version. I think minor version makes sense. For this, since it's the same functionality across both packages, you can use a single changeset file.